<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_friend</class>
 <widget class="QWidget" name="add_friend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="mini_butt">
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: green;
        border: none;
        border-radius: 10px;
        width: 20px;
        height: 20px;
        font-size: 14px;
        color: white;
    }
    
    QPushButton:hover {
        background-color: rgba(0, 128, 0, 0.8);
    }
    
    QPushButton:pressed {
        background-color: rgba(0, 128, 0, 1);
    }</string>
         </property>
         <property name="text">
          <string>➖</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_butt">
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: red;
        border: none;
        border-radius: 10px;
        width: 20px;
        height: 20px;
        font-size: 12px;
        color: white;
    }
    
    QPushButton:hover {
        background-color: rgba(255, 0, 0, 0.8);
    }
    
    QPushButton:pressed {
        background-color: rgba(255, 0, 0, 1);
    }</string>
         </property>
         <property name="text">
          <string>✖</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>请输入好友id:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="id_in"/>
     </item>
     <item>
      <widget class="PushButton" name="add_fri_butt">
       <property name="text">
        <string>添加好友</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
